web-ts/src/components/HeaderTitle.tsx

```diff
@@ -1,18 +1,15 @@
-// @flow
-
 import * as React from 'react'
-import styled, { type StyledComponent } from 'styled-components'
-import type { ThemeType } from 'build-configs/ThemeType'
-import dimensions from '../../theme/constants/dimensions'
+import styled from 'styled-components'
+import dimensions from '../constants/dimensions'
 
-type PropsType = {|
-  children: ?string
-|}
+type PropsType = {
+  children: string | undefined
```

Comment:
personally I prefer `children?: string`, just as it looks better. You decide.

Reply:
Will fix in #305